packages/slice-machine/test/src/modules/__fixtures__/serverState.ts

```diff
@@ -25,6 +25,7 @@ export const dummyServerState: Pick<
       PrismicUnsplash: "https://unsplash.prismic.io/",
       SliceMachineV1: "https://sm-api.prismic.io/v1/",
       RepositoryService: "https://repository.prismic.io/",
+      LocaleService: "https://locale.prismic.io/",
```

Comment:
⚠️ Should it be `locale.internal.prismic.io` here also?

Reply:
yes, I think so, thanks!

Reply:
updated